website/server/controllers/api-v3/groups.js

```diff
@@ -948,7 +955,12 @@ api.removeGroupMember = {
     }
 
     if (isInGroup) {
-      group.memberCount -= 1;
+      if (group.type === 'party') {
```

Comment:
can we add the same comment of above here?

Reply:
Added the comment.